reporter/src/main/kotlin/Main.kt

```diff
@@ -143,5 +144,11 @@ object Main {
         val scanRecord = scanRecordFile?.let {
             it.mapper().readValue(it, ScanRecord::class.java)
         } ?: throw IllegalArgumentException("Could not read scan record from '$scanRecordFile'.")
+
+        reportFormats.forEach {
+            when (it) {
+                ReportFormat.EXCEL -> ExcelReporter().generateReport(scanRecord, outputDir)
```

Comment:
How about instead associating each `ReporterFormat` with a function to generate the report, or directly add that function to the enum instance? But this is also something we could improve later.

Reply:
Good idea, but I prefer to improve this later.